pulse2percept/api.py

```diff
@@ -472,15 +532,21 @@ def plot_fundus(self, stim=None, ax=None, upside_down=True, annotate=True):
         elif hasattr(ax, 'set_axis_bgcolor'):
             ax.set_axis_bgcolor('black')
 
-        # Draw axon pathways
-        ax.plot(retina.dva2ret(self.ofl.jan_x[:, ::5]),
-                retina.dva2ret(self.ofl.jan_y[:, ::5]),
-                c=(0.5, 1, 0.5))
+        # Draw axon pathways: Need to regenerate because parfor returns axons
+        # out of orders - random sample doesn't look neat. Should be fast:
```

Comment:
Will this mess with us, if axons are generated in some way that uses non-default settings?

Reply:
In particular, do I understand correctly that the default value for `n_axons` here is *different* from the default value used above?

Reply:
Good point - I was able to pass all keyword arguments from `sim.ofl` into this code bit. The only difference is a smaller `n_axons`, so that the plot is not just all green.